integration/windows_hostprocess_test.go

```diff
@@ -127,12 +127,14 @@ func runHostProcess(t *testing.T, expectErr bool, image string, action hpcAction
 	action(t, cn, containerConfig)
 }
 
-func startAndTestContainer(t *testing.T, sb string, sbConfig *runtime.PodSandboxConfig, cnConfig *runtime.ContainerConfig) {
+func createStartStopAndRemoveTestContainer(t *testing.T, sb string, sbConfig *runtime.PodSandboxConfig, cnConfig *runtime.ContainerConfig) {
```

Comment:
This is a mouthful, I'd say runTestContainer as run implies create+start, but we're also explicitly stopping and removing.. Maybe `runAndRemoveContainer`?